Automattic/crowdsignal-plugin
# Fix icon load in Dashboard menu

The dashboard icon is loading at a delay because it is loading a font:

https://cloudup.com/c7Ti684dCVw

I've updated this to use an encoded svg following the guide here: https://developer.wordpress.org/reference/functions/add_menu_page/

I don't think we need the font files with this change so I've deleted them!

**To Test**

Load up the admin, the Crowdsignal logo should load properly and have the correct color based on active state.

<img width="160" alt="Screen Shot 2020-02-11 at 2 58 24 PM" src="https://user-images.githubusercontent.com/789137/74283384-01cf4400-4cdf-11ea-8de6-d54e5f8b69d9.png">
